src/class-parsely.php

```diff
@@ -1860,4 +1912,22 @@ public function parsely_is_user_logged_in() {
 	public function return_personalized_json() {
 
 	}
+
+	/**
+	 * Convert JSON-LD types to respective Parse.ly page types
+	 *
+	 * @see https://www.parse.ly/help/integration/metatags#field-description
+	 * @since 2.5.0
+	 *
+	 * @param  $type string
+	 * @return string
```

Comment:
```suggestion
	 * Convert JSON-LD type to respective Parse.ly page type.
	 *
	 * If it's one of the supported post types, then "post" will be returned.
	 * Otherwise, "index" is returned.
	 *
	 * @since 2.5.0
	 *
	 * @see https://www.parse.ly/help/integration/metatags#field-description
	 *
	 * @param $type string JSON-LD type
	 * @return string "post" or "index".
```